layouts/partials/footer.html

```diff
@@ -47,5 +47,13 @@ <h5>About</h5>
     </footer>
     <script src="{{ "/_/js/site.js" | relURL }}"></script>
     <script src="{{ "/_/js/vendor/highlight.js" | relURL }}"></script>
+    <script type="application/ld+json">
+        {
+        "@context": "http://schema.org/",
+        "@type": "Organization",
+        "url": "https://camel.apache.org/staging/",
+        "logo": "https://camel.apache.org/staging/_/img/logo-d.svg"
+        }
+      </script>
```

Comment:
```suggestion
    </script>
```

To make it align with the opening `<script>` tag.